go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/lightninglabs/aperture
 go 1.15
```

Comment:
i think we should bump this go version too?

Reply:
Good point, might as well!